Automattic/newspack-plugin
# feat(prompt settings): add prompt new placements and size options

### All Submissions:






### Changes proposed in this Pull Request:



https://github.com/Automattic/newspack-popups/pull/624 adds new placements for overlay prompts and
size option, we need to add them to the settings wizard

### How to test the changes in this Pull Request:

1. If https://github.com/Automattic/newspack-popups/pull/624 is not merged, please pull it.
2. Checkout this PR.
3. Add a new overlay prompt, and save it.
4. From the campaigns list, click on the `prompt settings` icon next to the new prompt.
5. Observe that you can set the new placements (top-left, top-right, ...) and the overlay size.
6. Change the overlay placement and size, and save the settings.
7. Edit the prompt, and notice that the new placement and size were saved from the previous step.

### Other information:
